Source/Core/Core/HW/DSPHLE/UCodes/AXVoice.h

```diff
@@ -428,13 +428,15 @@ void ProcessVoice(PB_TYPE& pb, const AXBuffers& buffers, u16 count, AXMixControl
 #define MIX_ON(C) (0 != (mctrl & MIX_##C))
 #define RAMP_ON(C) (0 != (mctrl & MIX_##C##_RAMP))
 
-  if (MIX_ON(L))
-    MixAdd(buffers.main_left, samples, count, &pb.mixer.main_left, &pb.dpop.left, RAMP_ON(L));
-  if (MIX_ON(R))
-    MixAdd(buffers.main_right, samples, count, &pb.mixer.main_right, &pb.dpop.right, RAMP_ON(R));
-  if (MIX_ON(S))
-    MixAdd(buffers.main_surround, samples, count, &pb.mixer.main_surround, &pb.dpop.surround,
-           RAMP_ON(S));
+  if (MIX_ON(MAIN_L))
+    MixAdd(buffers.main_left, samples, count, &pb.mixer.main_left, &pb.dpop.main_left,
+           RAMP_ON(MAIN_L));
+  if (MIX_ON(MAIN_R))
+    MixAdd(buffers.main_right, samples, count, &pb.mixer.main_right, &pb.dpop.main_right,
+           RAMP_ON(MAIN_R));
+  if (MIX_ON(MAIN_S))
+    MixAdd(buffers.main_surround, samples, count, &pb.mixer.main_surround, &pb.dpop.main_surround,
+           RAMP_ON(MAIN_S));
```

Comment:
Would you mind adding curlies in here (and the remainder of the method, which I can't mark for comments on here), since the contents often span more than one line?
```suggestion
  if (MIX_ON(MAIN_L))
  {
    MixAdd(buffers.main_left, samples, count, &pb.mixer.main_left, &pb.dpop.main_left,
           RAMP_ON(MAIN_L));
  }
  if (MIX_ON(MAIN_R))
  {
    MixAdd(buffers.main_right, samples, count, &pb.mixer.main_right, &pb.dpop.main_right,
           RAMP_ON(MAIN_R));
  }
  if (MIX_ON(MAIN_S))
  {
    MixAdd(buffers.main_surround, samples, count, &pb.mixer.main_surround, &pb.dpop.main_surround,
           RAMP_ON(MAIN_S));
  }
```